near-contract-standards/src/non_fungible_token/utils.rs

```diff
@@ -26,7 +26,7 @@ pub fn refund_approved_account_ids(
     refund_approved_account_ids_iter(account_id, approved_account_ids.keys())
 }
 
-pub fn refund_deposit(storage_used: u64) {
+pub fn refund_deposit(storage_used: u64, account_id: AccountId) {
```

Comment:
to avoid breaking changes, this should probably be a new function, and just have `refund_deposit` call this with the `predecessor_account_id`. Can think about deprecating this old function too if accepted